cdap-hbase-compat-1.1/src/main/java/co/cask/cdap/data2/replication/hbase11/LastWriteTimeObserver.java

```diff
@@ -67,8 +71,8 @@ public void postWALWrite(ObserverContext<? extends WALCoprocessorEnvironment> ct
               logKey.getTablename().toString(),
               logKey.getWriteTime(),
               logKey.getEncodedRegionName().toString());
-    if (tableUpdater != null) {
-      tableUpdater.updateTime(new String(logKey.getEncodedRegionName(), "UTF-8"), logKey.getWriteTime());
+    if (hBase11TableUpdater != null) {
+      hBase11TableUpdater.updateTime(new String(logKey.getEncodedRegionName(), "UTF-8"), logKey.getWriteTime());
```

Comment:
New region name format: [0.99+]
<tablename><startkey><regionIdTimestamp><encodedName>.
 where, <encodedName> is a hex version of the MD5 hash of
<tablename><startkey><regionIdTimestamp>. HRegionInfo.encodeRegionName() returns this hex version of the MD5. [https://github.com/apache/hbase/blob/master/hbase-client/src/main/java/org/apache/hadoop/hbase/HRegionInfo.java#L88]
For our use case, we just need any unique representation for a Region. From the above description of the regionName, it looks like UTF-8 should always be ok.